app/views/helpers/AssociatedCompaniesViewHelper.scala

```diff
@@ -51,15 +51,12 @@ object AssociatedCompaniesViewHelper extends ViewHelper {
         None
     }
 
-  def heading(a: AssociatedCompaniesParameter)(implicit messages: Messages): Html =
```

Comment:
Any specific reason to move heading classes to html from Helper? just curious

Reply:
> Any specific reason to move heading classes to html from Helper? just curious

So we want to copy the h1 (message) into the legend of radio button. When we get this value from helper it gets the message along with the html styling

Reply:
this method was already in the Helper, I agree with the question though, it seems like this could live comfortably within the template (unless there are specific unit tests on it), @ruthdas03 what do you think? remove from helper and make the pattern match the value of `@heading` in the template?

Reply:
oh I see there is a spec for this helper, and it's part of the updates, so that's probably a good reason to leave it in the helper! Sorry should have checked first